docs/sources/enterprise/usage-insights.md

```diff
@@ -1,6 +1,6 @@
 +++
-title = "Usage-insights"
-description = "Usage-insights"
+title = "Usage insights"
+description = "See who is using dashboards and how often"
```

Comment:
This sounds like it is identifying actual people. How about, "See how often dashboards are being used"

Reply:
I think the passive is warranted here because the point is to id the dashboards, not the people.